### DISCUSSION TITLE
IS it possible to disable anchor linking?

### DISCUSSION THREAD

**Discussion by AnastasiaTWW** [squidfunk/mkdocs-material#6449] [General]
COuld you please advise if there is a way to disable anchor linking for content tabs https://squidfunk.github.io/mkdocs-material/reference/content-tabs/?h=anchor#anchor-links?

**Comment by squidfunk** [comment]
No, it's not possible, because to my knowledge it only has upsides. Why would you want to do that?

**Reply by AnastasiaTWW** [reply]
We use anchor tracking https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#anchor-tracking and I can make it work together with content tab anchor linking. It may confuse users - they click a tab, see that the URL changes and scroll the page keeping in mind that they have the chosen tab reflected in the URL but it changes

**Reply by squidfunk** [reply]
Yes, when you scroll, the anchor link is updated, that's correct. Note that we're going to revisit the entire problem of "what element is active" when we tackle #4125. There're some comments in there that explain the problem.

**Reply by squidfunk** [reply]
On another note: without anchor links you will not ne able to link to tabs, e.g. from other parts of your documentation. Thus, it's actually a must-have feature.

**Reply by AnastasiaTWW** [reply]
Got it. Thank you for your reply!
